### PR TITLE
bank management

### PR DESCRIPTION
**Pull request by sona-podholova**
# TL;DR

Bank management

# Proof
image

# Merge request checklist

Please check if your merge request fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been reviewed and added/updated if needed (for bug fixes/features)
- [x] Build (`yarn build`) was run locally and any changes were pushed for both API and clients
- [x] API (`yarn api:dev`) runs locally and any fixes were made for failures
- [x] Admin Client (`cd clients/admin && yarn start`) runs locally and any fixes were made for failures

# Merge request type

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API  changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

# Before this merge request
no bank management

# How you fixed what was wrong
bank management added

### CODE REVIEW

File: clients/admin/src/components/add-bank/AddOrEditBank.svelte [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+</script>
+
+<div class="mt-10 sm:mt-0">
+    <div class="md:gap-6">
+        <div class="mt-5 md:mt-0">
+            <div class="shadow overflow-hidden sm:rounded-md">
+                <div class="px-4 py-5 bg-white sm:p-6">
+                    <SimpleTextBox
+                        form={myForm}
+                        title="Display name"
```
Comment (davidpodhola) [inline clients/admin/src/components/add-bank/AddOrEditBank.svelte:61]: this should be translated

**Review by davidpodhola — APPROVED** [review]
